x-pack/plugins/canvas/public/components/workpad_header/control_settings/control_settings.js

```diff
@@ -36,45 +36,50 @@ const getRefreshInterval = (val = '') => {
   }
 };
 
-export const RefreshControl = ({ inFlight, setRefreshInterval, refreshInterval, doRefresh }) => {
+export const ControlSettings = ({ inFlight, setRefreshInterval, refreshInterval, doRefresh }) => {
   const setRefresh = val => setRefreshInterval(getRefreshInterval(val));
 
+  const disableInterval = () => {
+    setRefresh(0);
+  };
+
   const popoverButton = handleClick => (
     <EuiButtonEmpty onClick={handleClick}>
-      <div style={{ display: 'flex', alignItems: 'center' }}>Refresh</div>
+      <EuiButtonIcon iconType="gear" aria-label="Control settings" />
     </EuiButtonEmpty>
   );
 
-  const autoRefreshControls = (
+  return (
     <Popover
       id="auto-refresh-popover"
       button={popoverButton}
-      panelClassName="canvasRefreshControl__popover"
+      anchorPosition="rightUp"
+      panelClassName="canvasControlSettings__popover"
     >
       {({ closePopover }) => (
-        <div>
-          <AutoRefreshControls
-            inFlight={inFlight}
-            refreshInterval={refreshInterval}
-            setRefresh={val => {
-              setRefresh(val);
-              closePopover();
-            }}
-            doRefresh={doRefresh}
-            disableInterval={() => {
-              setRefresh(0);
-              closePopover();
-            }}
-          />
-        </div>
+        <EuiFlexGroup>
```

Comment:
Using flex here so that I can put more setting next to this in a new `<EuiFlexItem>`